pkg/workceptor/command.go

```diff
@@ -239,6 +240,35 @@ func (cw *commandUnit) Start() error {
 	return cw.runCommand(cmd)
 }
 
+// isOrphaned checks if command runner for this unit exists
+func (cw *commandUnit) isOrphaned() (bool, error) {
+	pid := cw.Status().ExtraData.(*commandExtraData).Pid
+	p, err := process.NewProcess(int32(pid))
+	if err != nil {
+		// process does not exist
+		return true, err
+	}
+
+	cmd, err := p.CmdlineSlice()
+	if err != nil {
+		return true, err
```

Comment:
Im worried here that if receptor may not have privilege to view the cmd line slice, do you know if their are any restrictions on which type of user can see this data?

Reply:
e.g. do we need sudo? do we need to be in a user group etc.